dashboard/model_layer.py

```diff
@@ -173,6 +173,8 @@ def enrich_session_info(se):
     se.started = humanize.naturaltime(session_time.total_seconds())
     se.status = 'Ongoing' if se.is_active() else 'Completed'
     se.shortened_node_key = helpers.shorten_node_key(se.node_key)
+    se.client_country_string = \
```

Comment:
And where is `ProviderCountry` in session?

Reply:
It's hard to understand the scope of this task since it doesn't have Jira task.
But for me, this looks like it can be done later in a separate PR :)

Reply:
created separate task MYST-440